labelbox/data/serialization/ndjson/objects.py

```diff
@@ -96,7 +96,8 @@ def to_common(self) -> Line:
     def from_common(cls, line: Line,
                     classifications: List[ClassificationAnnotation], name: str,
                     feature_schema_id: Cuid, extra: Dict[str, Any],
-                    data: Union[ImageData, TextData]) -> "NDLine":
+                    data: Union[ImageData,
+                                TextData], confidence: float) -> "NDLine":
```

Comment:
Won't this break if confidence isn't set?

```
        optional_kwargs = {}
        if (annotation.confidence):
            optional_kwargs['confidence'] = annotation.confidence
...
       
        return obj.from_common(annotation.value, subclasses, annotation.name,
                               annotation.feature_schema_id, annotation.extra,
                               data, **optional_kwargs)
```
There is no positional argument for confidence or default

Reply:
correct, fixed